modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/exp/agg/GroupKey.java

```diff
@@ -19,13 +19,16 @@
 
 import static org.apache.ignite.internal.util.ArrayUtils.OBJECT_EMPTY_ARRAY;
 
+import java.io.Serializable;
 import java.util.Arrays;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * A fixed length group key used by execution node.
  */
-public class GroupKey {
+public class GroupKey implements Serializable {
```

Comment:
GroupKey must not be serializable

Reply:
but we send it into the wire, ItSetOpTest#testIntersectEmpty

Reply:
I'm wondering how it works without implementing Serializable... Anyway, the ticket that addresses mentioned place has been merged to main moment ago. I hope this won't be a blocker for you anymore

Reply:
ok, merging fresh main helps me here.